docs/api.yml

```diff
@@ -255,6 +257,178 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GithubAuthIssue'
+  /{username}/pokemon:
+    get:
+      tags:
+        - pokemon
+      summary: Get all of the pokemon in the list
+      description: Returns an array of all of the existing pokemon in the list
+      operationId: getAllPokemon
+      parameters:
+        - name: username
+          in: path
+          description: your github username
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Pokemon'
+        '401':
+          description: The username provided is not part of the boolean-uk github organisation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GithubAuthIssue'
+    post:
+      tags:
+        - pokemon
+      summary: Add a new pokemon to the list
+      description: Add a new pokemon to the list
+      operationId: addPokemon
+      parameters:
+        - name: username
+          in: path
+          description: your github username
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Create a new pokemon in the list
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Pokemon'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/Pokemon'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/Pokemon'
+        required: true
+      responses:
+        '201':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Pokemon'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/Pokemon'
+        '400':
+          description: bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequest'
+        '401':
+          description: The username provided is not part of the boolean-uk github organisation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GithubAuthIssue'
+    put:
```

Comment:
```suggestion
  /{username}/pokemon/{pokemonId}:
    put:
```

Reply:
This should put the pokemon id in the URL properly while using swagger

Reply:
Noice, fixed